server/src/yourturn.js

```diff
@@ -16,13 +19,11 @@ require('./monitoring/monitoring');
 require('./env');
 
 // this is the actual server code
-var fs = require('fs'),
-    express = require('express'),
+var express = require('express'),
```

Comment:
I first added appdynamics config to yourturn.js. Therefore i had to move fs require to the top. I then realized the monitoring.js which encapsulates any monitoring config and i moved it over there.